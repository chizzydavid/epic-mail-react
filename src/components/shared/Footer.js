import React from 'react'

const Footer = () => {
  return (
    <React.Fragment>
      <footer>    
        <div className="" id="footer">
          <p>&copy; 2019, <strong>EPIC MAIL</strong>
          <br/>
          All trademarks and registered trademarks appearing on 
          this site are the property of their respective owners.</p>      
        </div>
      </footer>
    </React.Fragment>
  )
}

export default Footer
