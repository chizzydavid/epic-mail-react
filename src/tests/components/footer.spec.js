import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/shared/Footer';

describe('Component: Footer', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
