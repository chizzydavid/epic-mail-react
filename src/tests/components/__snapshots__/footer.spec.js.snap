// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer should render correctly 1`] = `ShallowWrapper {}`;
